src/backend/db/products.js

```diff
@@ -1,30 +1,88 @@
 import { v4 as uuid } from "uuid";
-
-/**
- * Product Database can be added here.
- * You can add products of your wish with different attributes
- * */
+import hoodie1 from "../../assets/products/hoodies/hoodie1.png";
+import hoodie2 from "../../assets/products/hoodies/hoodie2.png";
+import hoodie3 from "../../assets/products/hoodies/hoodie3.png";
+import hoodie4 from "../../assets/products/hoodies/hoodie4.png";
+import hoodie5 from "../../assets/products/hoodies/hoodie5.png";
+import hoodie6 from "../../assets/products/hoodies/hoodie6.png";
```

Comment:
Suggestion- Instead of importing every image individually, you can do do a barrel export to Import every file from a single source. Rest everything seems good.

Reply:
yes, will do that! Thanks